src/rtc/rtc_s1_single_job.py

```diff
@@ -71,26 +73,145 @@ def update_mosaic_boundaries(mosaic_geogrid_dict, geogrid):
 
 
 
+def _save_browse(burst_imagery_list, browse_image_filename,
+                 pol_list, browse_image_height, browse_image_width,
+                 temp_files_list, scratch_dir, logger):
+    """Create and save a browse image for the RTC-S1 product
 
+       Parameters
+       ----------
+       burst_imagery_list : list(str)
+           List of burst imagery files (one file for each polarization channel)
+       browse_image_filename : str
+           Output browse file
+       pol_list : list(str)
+           List of polarization channels
+       browse_image_height : int
+           Browse image height
+       browse_image_width : int
+           Browse image width
+       scratch_dir : str
+           Directory for temporary files
+       temp_files_list: list (optional)
+           Mutable list of temporary files. If provided,
+           paths to the temporary files generated will be
+           appended to this list.
+       logger : loggin.Logger
+           Logger
+    """
 
+    BROWSE_IMAGE_MAX_PERCENTILE = 97
 
+    logger.info(f'creating browse image: {browse_image_filename}')
+
+    n_images = len(burst_imagery_list)
+
+    if n_images == 1:
+        expected_pol_order = pol_list
+    elif n_images == 2 and 'HH' in pol_list:
+        expected_pol_order = ['HH', 'HV']
+    elif n_images == 2:
+        expected_pol_order = ['VV', 'VH']
+    else:
+        expected_pol_order = ['HH', 'HV', 'VV']
```

Comment:
```suggestion
    elif n_images == 3:
        expected_pol_order = ['HH', 'HV', 'VV']
    else:
        raise ValueError('Unexpected number of images '
                         f'in  the burst imagery list (n_images) ')
```

Reply:
Here, the idea is to use `['HH', 'HV', 'VV']` even if you have 3 or more images (e.g., quad-pol). But I can raise an error if there's more than 4.